scripts/scrape-translations.mjs

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import path from 'path';
+import { execSync } from 'child_process';
+
+const constantsPackagePath = path.join(import.meta.dirname, '..', 'packages', 'constants');
+
+const enumTranslationsScript = path.join(
+  constantsPackagePath,
+  'scripts',
+  'printTranslatedEnums.ts',
+);
```

Comment:
At this point we could just... import enums directly and run this whole thing in ts

and maybe produce JSON directly

Reply:
Do we still care about generating the translations delta between a version and the previous version?

Reply:
We need some way of surfacing new translation strings but I'm not sure if that's the best tbh